_config/products.yml

```diff
@@ -728,13 +745,14 @@ jbt_core:
             md5_url: http://sourceforge.net/projects/jboss/files/JBossTools/jbosstools4.3.0.x/jbosstools-4.3.0.Alpha1_2015-02-15_14-44-55-B13-browsersim-standalone.zip.MD5
           - name: Release Notes
             url: http://tools.jboss.org/documentation/whatsnew/index.html
-      4.3.0.Nightly:
-        update_site_url: http://download.jboss.org/jbosstools/updates/nightly/mars/
+      4.3.x.Nightly:
+        update_site_url: http://download.jboss.org/jbosstools/mars/snapshots/updates/
```

Comment:
http://download.jboss.org/jbosstools/mars/snapshots/updates/
currently points to 
http://download.jboss.org/jbosstools/mars/staging/updates/core/4.3.0.CR2b
http://download.jboss.org/jbosstools/mars/staging/updates/central/4.3.0.CR2b/
They don't look like right places for core/central nightly updatesites?